anchore_engine/analyzers/modules/32_java_packages.py

```diff
@@ -27,27 +17,44 @@
 try:
     config = anchore_engine.analyzers.utils.init_analyzer_cmdline(sys.argv, analyzer_name)
 except Exception as err:
-    print str(err)
+    print(str(err))
     sys.exit(1)
 
 imgname = config['imgid']
 imgid = config['imgid_full']
 outputdir = config['dirs']['outputdir']
 unpackdir = config['dirs']['unpackdir']
 
-def process_java_archive(prefix, filename, inZFH):
+
+def parse_properties(file):
+    """
+    Parses the given file using the Java properties file format.
+    Lines beginning with # are ignored.
+    :param file: an open iterator into the file
+    :return: the properties in the file as a dictionary
+    """
+    props = {}
+    for line in file:
+        if not re.match("\s*(#.*)?$", line):
+            kv = line.split('=')
+            key = kv[0].strip()
+            value = '='.join(kv[1:]).strip()
+            props[key] = value
+    return props
+
+
+def process_java_archive(prefix, filename, inZFH=None):
     ret = []
 
-    
     fullpath = '/'.join([prefix, filename])
 
     jtype = None
     patt = re.match(java_library_file, fullpath)
     if patt:
         jtype = patt.group(1)
     else:
-        return([])
-    name = re.sub("\."+jtype+"$", "", fullpath.split("/")[-1])
+        return []
```

Comment:
It might be best to leave out these kinds formatting changes (and the print import from \_\_future\_\_, etc.) unless the maintainers are fine with it.

Reply:
I added the future import after talking with them about Python 3, and they're planning to upgrade in the near future, so I figured I could be helpful in the file I'm changing.

Reply:
The formatting change is not a problem. We're pretty far from PEP-8 now, and don't have a rigorous formatting guideline currently--it will be PEP-8 once we have cycles to do it.